src/sample/Controller.java

```diff
@@ -1,4 +1,82 @@
 package sample;
 
-public class Controller {
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable {
+
+    ObservableList<Contact> contacts = FXCollections.observableArrayList();
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField text;
+
+    @FXML
+    TextField text1;
+
+    @FXML
+    TextField text2;
+
+
+    public void addcontact() {
+        //System.out.println("addItem");
+        contacts.add(new Contact(text.getText(), text1.getText(), text2.getText()));
```

Comment:
You need a line above line 32 to stop the user from adding a contact with a blank field.  Everything else is correct.  Figure out that line and resubmit.